tests/test_schema_validation.py

```diff
@@ -55,6 +55,18 @@ def test_schemas(self):
             for error in errors:
                 logger.error(error)
 
+    def test_invalid_answer_range(self):
+
+        file = 'schemas/test_invalid_answer_range.json'
+
+        json_file = open(os.path.join(os.path.dirname(os.path.realpath(__file__)), file), encoding='utf8')
+        json_to_validate = load(json_file)
```

Comment:
You can probably make these 2 lines into a reusable function for the rest of the tests here?